src/lsp/MessageHandlers.re

```diff
@@ -576,7 +564,7 @@ let handlers: list((string, (state, Json.t) => result((state, Json.t), string)))
       let%opt () = Filename.check_suffix(uri, ".re") ? Some(()) : None;
 
       let pos = Utils.cmtLocFromVscode(pos);
-      let%opt (location, loc) = References.locForPos(~extra, pos);
+      let%opt (_, loc) = References.locForPos(~extra, pos);
```

Comment:
could this be e.g. `_location` instead of `_`? Same for the other places where there's a positional variable that's no longer used